### PR TITLE
Added code to filter basic unsupported datatypes in export data from source and export data from target for PG and YB

### PR DESCRIPTION
**Pull request by ShivanshGahlot**

### CODE REVIEW

File: yb-voyager/src/srcdb/postgres.go [hunk 1]
```diff
@@ -415,11 +417,53 @@ func (pg *PostgreSQL) FilterEmptyTables(tableList []*sqlname.SourceName) ([]*sql
...
+	var columns, dataTypes, dataTypesOwner []string
+	query := fmt.Sprintf(`SELECT a.attname AS column_name, t.typname AS data_type, rol.rolname AS data_type_owner 
+						FROM pg_attribute AS a JOIN pg_type AS t ON t.oid = a.atttypid 
+						JOIN pg_class AS c ON c.oid = a.attrelid 
+						JOIN pg_namespace AS n ON n.oid = c.relnamespace 
+						JOIN pg_roles AS rol ON rol.oid = t.typowner 
+						WHERE c.relname = '%s' AND n.nspname = '%s' AND a.attname NOT IN ('tableoid', 'cmax', 'xmax', 'cmin', 'xmin', 'ctid');`, tableName.ObjectName.Unquoted, tableName.SchemaName.Unquoted)
+	rows, err := pg.db.Query(context.Background(), query)
+	if err != nil {
+		utils.ErrExit("error in querying(%q) source database for table columns: %v\n", query, err)
```
Comment (makalaaneesh) [inline yb-voyager/src/srcdb/postgres.go:429]: As a general practice, we shouldn't be using utils.ErrExit and rely on returning error objects. Quite a bit of the existing code is written with utils.ErrExit, but let's try our best to not add more. 

If possible, fix this func and GetColumnsWithSupportedTypes in the interface.

File: yb-voyager/src/srcdb/yugabytedb.go [hunk 2]
```diff
@@ -359,11 +361,54 @@ func (yb *YugabyteDB) FilterEmptyTables(tableList []*sqlname.SourceName) ([]*sql
 }
 
 func (yb *YugabyteDB) GetTableColumns(tableName *sqlname.SourceName) ([]string, []string, []string) {
-	return nil, nil, nil
+	var columns, dataTypes, dataTypesOwner []string
+	query := fmt.Sprintf(`SELECT a.attname AS column_name, t.typname AS data_type, rol.rolname AS data_type_owner
```
Comment (makalaaneesh) [inline yb-voyager/src/srcdb/yugabytedb.go:365]: If it's the same query template as that of PG, try defining it as a variable ?

File: yb-voyager/src/srcdb/yugabytedb.go [hunk 3]
```diff
@@ -34,6 +34,8 @@ import (
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils/sqlname"
 )
 
+var yugabyteUnsupportedDataTypesForDbzm = []string{"BOX", "CIRCLE", "LINE", "LSEG", "PATH", "PG_LSN", "POINT", "POLYGON", "TSQUERY", "TSVECTOR", "TXID_SNAPSHOT"}
```
Comment (makalaaneesh) [inline yb-voyager/src/srcdb/yugabytedb.go:37]: Shall we add the GEOMETRY, GEOGRAPHY to this list as well? I know we didn't test, but doesn't hurt to put it?

File: yb-voyager/src/srcdb/yugabytedb.go (hunk 2 above)
Comment (ShivanshGahlot) [reply yb-voyager/src/srcdb/yugabytedb.go:365]: Nope they are a little different

File: yb-voyager/src/srcdb/postgres.go (hunk 1 above)
Comment (ShivanshGahlot) [reply yb-voyager/src/srcdb/postgres.go:429]: Done

File: yb-voyager/src/srcdb/yugabytedb.go (hunk 2 above)
Comment (ShivanshGahlot) [reply yb-voyager/src/srcdb/yugabytedb.go:365]: Done

File: yb-voyager/src/srcdb/yugabytedb.go (hunk 3 above)
Comment (ShivanshGahlot) [reply yb-voyager/src/srcdb/yugabytedb.go:37]: Done

**Pull request closed without merging**